pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -587,6 +600,59 @@ var _ = Describe("Pod Network", func() {
 				api.NewDefaulter(runtime.GOARCH).SetObjectDefaults_Domain(domain)
 				TestPodInterfaceIPBinding(vm, domain)
 			})
+			It("should define a new DhcpConfig bind to a bridge with Istio enabled and create a default nat rule using nftables", func() {
```

Comment:
Not related to your PR. But the text in the `It`s here is horrible. How `DHCPConfig` is related here?
@maiqueb

Reply:
hm, my best guess is this was an unwanted side effect of using the IDE to rename from VIF to DhcpConfig 0_o

"should define a new pod interface bound to a ...." would be the correct thing.

Reply:
@rhrazdil can you please fix the description of those tests (yours and the other similar ones), to something that will describe better what they actually do?

Reply:
@AlonaKaplan ops, I missed your comment here. Will do

Reply:
https://github.com/kubevirt/kubevirt/pull/5461/commits/d84e9decd176872433196ced9e28396914f4ac5e